plugin.cmake

```diff
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
```

Comment:
Will this work properly with older versions of CMake that don't have this policy?